export/tests/requirements/111.txt

```diff
@@ -0,0 +1,2 @@
+Django~=1.11.15
```

Comment:
These requirements can be listed directly in the `tox.ini` file. No need for separate files for two requirements.

Reply:
See https://github.com/praekelt/django-object-tools/blob/develop/tox.ini as an example